### PR TITLE
Don't return non-200 responses

### PR DESCRIPTION
**Pull request by swankjesse**

### CODE REVIEW

File: zipline-loader/build.gradle.kts [hunk 1]
```diff
@@ -48,6 +48,7 @@ kotlin {
       dependsOn(engineMain)
       dependencies {
         implementation(libs.okHttp.core)
+        implementation(libs.okHttp.mockWebServer)
```
Comment (JakeWharton) [inline zipline-loader/build.gradle.kts]: This should be in jniTest

File: zipline-loader/src/jvmTest/kotlin/app/cash/zipline/loader/OkHttpZiplineHttpClientTest.kt [hunk 2]
```diff
@@ -0,0 +1,70 @@
+/*
```
Comment (JakeWharton) [inline zipline-loader/src/jvmTest/kotlin/app/cash/zipline/loader/OkHttpZiplineHttpClientTest.kt]: Could move to jniTest to run on both JVM and Android

**Review by JakeWharton — APPROVED** [review]

File: zipline-loader/build.gradle.kts (hunk 1 above)
Comment (swankjesse) [reply zipline-loader/build.gradle.kts]: ooooh good catch. Definitely that was my intent

File: zipline-loader/src/jvmTest/kotlin/app/cash/zipline/loader/OkHttpZiplineHttpClientTest.kt (hunk 2 above)
Comment (swankjesse) [reply zipline-loader/src/jvmTest/kotlin/app/cash/zipline/loader/OkHttpZiplineHttpClientTest.kt]: Fixed.

**Pull request merged**